src/signals/incident/components/form/ContainerSelect/components/ContainerEditList/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useCallback } from 'react';
+import type { FunctionComponent } from 'react';
+import styled from 'styled-components';
+
+import { themeSpacing } from '@amsterdam/asc-ui';
+import { Close } from '@amsterdam/asc-assets';
+
+import Button from 'components/Button';
+import type { FeatureType, Item } from '../../types';
+
+const DEFAULT_ICON_SIZE = 40;
+const REMOVE_BUTTON_SIZE = 32;
+const REMOVE_ICON_SIZE = 12;
+const NAME = 'containerEditList';
```

Comment:
Moving the data-testid to a constant does not fit the coding style used in this project, the cases when a variable is used are when they are passed as props and have some other usage besides testid. Shell we keep consistency here?